ZukoAzula2016/src/org/team1540/zukoazula/Shooter.java

```diff
@@ -39,6 +39,20 @@ public static void setup() throws ExtendedMotorFailureException {
         BooleanInput fire = ZukoAzula.controlBinding.addBoolean("Shooter Fire");
         BooleanInput spinup = warmup.or(fire).or(actuallyFiring);
 
+        EventInput intakeArmRollerForward = ZukoAzula.controlBinding.addEvent("Intake Arm Rollers Forward");
+        EventInput intakeArmRollerBackward = ZukoAzula.controlBinding.addEvent("Intake Arm Rollers Backward");
+        EventInput intakeArmRollerStop = ZukoAzula.controlBinding.addEvent("Intake Arm Rollers Stop");
+
+        FloatCell rollerOutput = new FloatCell();
+        rollerOutput.setWhen(1, intakeArmRollerForward);
+        rollerOutput.setWhen(0, intakeArmRollerStop);
+        rollerOutput.setWhen(-1, intakeArmRollerBackward);
+
+        BooleanCell usingIntake = new BooleanCell();
+        usingIntake.setTrueWhen(intakeArmRollerBackward.or(intakeArmRollerForward));
+        usingIntake.setFalseWhen(intakeArmRollerStop.or(spinup.onRelease()));
+        rollerSpeed.attach(rollerArb.addBehavior("Using Intake", usingIntake), rollerOutput);
```

Comment:
... because they're not separate? I'm pretty sure you need to run the ball indexer as part of intaking. And if you want to spit a ball back out, the indexer also has to run.